setup.py

```diff
@@ -0,0 +1,29 @@
+import setuptools
```

Comment:
A common template I've seen with setup tools is to split the functionality into `setup.py` for code and `setup.cfg` for configuration.

Reply:
Good idea! Does 0cda163674a make sense as a way to do this?

Reply:
Great